kotsadm/web/src/components/apps/AppDetailPage.jsx

```diff
@@ -99,19 +101,29 @@ class AppDetailPage extends Component {
     this.state.getAppJob.stop();
   }
 
-  makeCurrentRelease = async (upstreamSlug, sequence) => {
-    try {
-      await fetch(`${window.env.API_ENDPOINT}/app/${upstreamSlug}/sequence/${sequence}/deploy`, {
-        headers: {
-          "Authorization": Utilities.getToken(),
-          "Content-Type": "application/json",
-        },
-        method: "POST",
+  makeCurrentRelease = (upstreamSlug, version) => {
+    this.setState({ makingCurrentReleaseErrMsg: "" });
+    fetch(`${window.env.API_ENDPOINT}/app/${upstreamSlug}/sequence/${version.sequence}/deploy`, {
+      headers: {
+        "Authorization": Utilities.getToken(),
+        "Content-Type": "application/json",
+      },
+      method: "POST",
+    })
+      .then(async (res) => {
+        if (!res.ok) {
+          const makingCurrentRelease = await res.json();
+          this.setState({ makingCurrentReleaseErrMsg: `Unable to deploy release ${version.title}, sequence ${version.sequence}: ${makingCurrentRelease.error}` });
```

Comment:
same here, there's no `error` key on the `res` object. might wanna use the status code.